requirements/constraints.txt

```diff
@@ -45,9 +45,18 @@ minio==7.0.4
 docker-compose==1.29.1
 
 
-# tomli<2.0.0,>=0.2.6 (from black==21.12b0->-r requirements/../../../requirements/devenv.txt (line 19))
-tomli<2.0.0
+# Could not find a version that matches typing-extensions==3.10.0.2,>=3.10,>=4.0 (from -c requirements/_base.txt (line 392))
+# Tried: 3.6.2, 3.6.2, 3.6.2.1, 3.6.2.1, 3.6.5, 3.6.5, 3.6.6, 3.6.6, 3.7.2, 3.7.2, 3.7.4, 3.7.4, 3.7.4.1, 3.7.4.1, 3.7.4.2, 3.7.4.2, 3.7.4.3, 3.7.4.3, 3.10.0.0, 3.10.0.0, 3.10.0.1, 3.10.0.1, 3.10.0.2, 3.10.0.2, 4.0.0, 4.0.0, 4.0.1, 4.0.1, 4.1.0, 4.1.0, 4.1.1, 4.1.1
+# There are incompatible versions in the resolved dependencies:
+#   typing-extensions==3.10.0.2 (from -c requirements/_base.txt (line 392))
+#   typing_extensions>=4.0 (from aioitertools==0.10.0->aiobotocore[boto3]==2.1.2->aioboto3==9.4.0->-r requirements/_test.in (line 29))
+#   typing-extensions>=3.10 (from astroid==2.9.3->pylint==2.12.2->-r requirements/_test.in (line 46))
+#   typing-extensions>=3.10.0 (from pylint==2.12.2->-r requirements/_test.in (line 46))
+#   typing-extensions>=3.10.0 (from bokeh==2.4.2->-r requirements/_test.in (line 38))
+aioitertools<=0.8.0
 
+# issues with pinned pyyaml version in 0.3.2>=, 0.5.x should be installable
+openapi-spec-validator<=0.3.1
```

Comment:
are there no side effect in downgrading that library so much?

Reply:
Tests look OK after this. I think it's not used in any core components of the platform. @pcrespov should know more better.

Reply:
well it is used all over the place in the webserver/storage/director-v0

Reply:
it is not a good policy to downgrade. There must be another way by limiting other library. Let us check this toghether offline one more time